core/domain/stats_domain.py

```diff
@@ -1486,6 +1486,10 @@ class StateAnswersCalcOutput(python_utils.OBJECT):
     state answers.
     """
 
+    # NOTE TO DEVELOPERS: The method save() was removed in #13021 as part of the
+    # migration to Apache Beam. Please refer to that PR if you need to reinstate
+    # it.
```

Comment:
I don't think we should promote adding models in the domain layer, so I would suggest removing this comment.

Reply:
Done.